.github/workflows/build-package.yml

```diff
@@ -94,60 +89,3 @@ jobs:
         with:
           name: ${{ inputs.artifact }}
           path: ./build/osconfig_*
-
-  signing:
```

Comment:
No longer signing?

Reply:
I plan to add the signing job back in a follow up PR as a separate workflow. Currently we are not performing signing in GitHub since packages are not yet published from here.

I will integrate this signing job into the changes for publishing packages (nightly/prod).